components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java

```diff
@@ -134,6 +135,16 @@ public void handleEvent(Event event) throws IdentityEventException {
         String accountLockTime = "0";
         int maximumFailedAttempts = 0;
         double unlockTimeRatio = 1;
+        String adminPasswordResetAccountLockNotificationProperty = IdentityUtil.getProperty(
+                AccountConstants.ADMIN_FORCE_PASSWORD_RESET_ACCOUNT_LOCK_NOTIFICATION_ENABLE_PROPERTY);
+        boolean adminForcePasswordResetLockNotificationEnabled =
+                !(adminPasswordResetAccountLockNotificationProperty == null) ||
```

Comment:
This will return true ... if the property is false.

Reply:
Boolean.FALSE.toString().equals("adminPasswordResetAccountLockNotificationProperty");